### PR TITLE
Add more exciting title to the landing page

### PR DESCRIPTION
**Pull request by MattSegal**
Hey reviewer!!! :) :)

I made this pull request so I can make the title more fun lol

### CODE REVIEW

File: app/web/templates/web/landing.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 <div id="landing-hero">
```
Comment (NickMakeThing) [inline app/web/templates/web/landing.html:8]: yes

File: app/web/templates/web/landing.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
...
     <div class="container wide">
-        <h1> Get free, ongoing legal<br/> assistance in 2 working days</h1>
```
Comment (NickMakeThing) [inline app/web/templates/web/landing.html:10]: dfsf

Comment (NickMakeThing) [reply app/web/templates/web/landing.html:10]: dfsdfds

**Review by NickMakeThing — COMMENTED** [review]
dfdsfdsfdsf

File: app/web/templates/web/landing.html (hunk 2 above)
Comment (MattSegal) [reply app/web/templates/web/landing.html:10]: I agree yeah

**Review by NickMakeThing — APPROVED** [review]

**Pull request merged**